src/gtk/textctrl.cpp

```diff
@@ -984,6 +992,16 @@ void wxTextCtrl::SetWindowStyleFlag(long style)
     if ( (style & wxTE_PROCESS_ENTER) != (styleOld & wxTE_PROCESS_ENTER) )
         GTKSetActivatesDefault();
 
+    if ( (style & wxTE_PROCESS_TAB) != (styleOld & wxTE_PROCESS_TAB) )
+    {
+        if ( IsMultiLine() )
+        {
+            gtk_text_view_set_accepts_tab(GTK_TEXT_VIEW(m_text),
+                                          style & wxTE_PROCESS_ENTER);
```

Comment:
This probably works, but since `gboolean` is really `int` I think it would be better to explicitly pass the value as a `bool`: `(style & wxTE_PROCESS_TAB) != 0`

Reply:
Oh, and you used `wxTE_PROCESS_ENTER`, but I think you meant `wxTE_PROCESS_TAB`?

Reply:
Thanks for the review and especially for noticing this typo, I've updated it to take your comments into account.